lib/filepath_to_url.js

```diff
@@ -0,0 +1,36 @@
+const url = require('url');
+
+if (url.pathToFileURL) {
+  module.exports = {
+    pathToFileURL: url.pathToFileURL,
+    fileURLToPath: url.fileURLToPath
+  };
+} else {
+  // Node 8 does not have nice url methods.
+  // Polyfill should match DevTools frontend behavior,
+  // otherwise breakpoints will not work.
+  function pathToFileURL(fileSystemPath) {
```

Comment:
Are you sure you want to implement this yourself? There's ought to be an NPM module for this.

Reply:
It should match DevTools implementation. I took implementation from https://cs.chromium.org/chromium/src/third_party/blink/renderer/devtools/front_end/common/ParsedURL.js?rcl=4cc12dbe644fc954f9bd6a5badb95d27f04a458e&l=95 .